src/Dapr.Client/DaprClient.cs

```diff
@@ -313,6 +313,21 @@ public HttpRequestMessage CreateInvokeMethodRequest<TRequest>(string appId, stri
         /// <param name="cancellationToken">A <see cref="CancellationToken" /> that can be used to cancel the operation.</param>
         /// <returns>A <see cref="Task{T}" /> that will return the value when the operation has completed.</returns>
         public abstract Task<bool> CheckHealthAsync(CancellationToken cancellationToken = default);
+
+        /// <summary>
+        /// Perform health-check of Dapr sidecar's outbound APIs. Return 'true' if the sidecar is health. Otherwise false.
+        /// </summary>
+        /// <param name="cancellationToken">A <see cref="CancellationToken" /> that can be used to cancel the operation.</param>
+        /// <returns>A <see cref="Task{T}" /> that will return the value when the operation has completed.</returns>
+        public abstract Task<bool> CheckOutboundHealthAsync(CancellationToken cancellationToken = default);
```

Comment:
What's the difference between this and `CheckHealthAsync` - how will users know which one is appropriate?

Reply:
Added to the comment explaining when to use one over the other. Essentially this is the one you want to hit when Dapr/your app are starting. It signifies that the endpoints that Dapr provides are up and ready. The other health check signifies when everything is up, including your application.

I'm also going to submit a docs issue in `dapr/docs` cause it's not referenced in the API Reference.

Reply:
Sounds good, thanks for adding the description.